src/Http/View/Composers/JavascriptComposer.php

```diff
@@ -46,6 +47,7 @@ public function compose(View $view)
             'locale' => config('app.locale'),
             'permissions' => $this->permissions($user),
             'hasLicenseBanner' => $licenses->invalid() || $licenses->requestFailed(),
+            'charmap' => ASCII::charsArray(),
```

Comment:
Will this support different dialects, like `fr_AT` over `fr`?

Reply:
Yeah, looks like it.

![image](https://user-images.githubusercontent.com/105211/158254913-5cc88d7d-c378-4511-a370-ad51ebdf8c12.png)